### ISSUE TITLE
Tissue concentration could not be further increased by increase in uptake transporter

### ISSUE THREAD

**Issue by yishuanwu** [Open-Systems-Pharmacology/Forum#519]
Hi, thank you again for your help in advance. 

I have an enzyme that is a homodimer of ~100 kD and is a substrate of the mannose-6-phosphate receptor that is dosed by IV infusion. 

I fitted the mice concentration data reasonably using mannose-6-phosphate receptor (uptake) based on observed tissue data, and intracellular + interstitial peptide hydrolysis assumed to be the same in all organs and plasma, except for in the liver, which was underpredicted by 8x:

![image](https://user-images.githubusercontent.com/13445680/81213190-0769bc80-8fa4-11ea-8d03-570dba744622.png)

Further increase in the liver uptake does not increase the liver concentration. Sensitivity analysis shows that the clearance process, and hematocrit/muscle volume/liver blood flow affects liver Cmax, however, these processes are not enough to explain the high Cmax, and I'd like to avoid changing PK-Sim mice default if possible. I looked into tissue binding, but because my drug does not readily permeate out of the cell, it does not seem to be useful to investigate. 

There are other very similar M6PR substrates, namely sulfamidase and beta-glucuronidase, that also have a very similar tissue distribution (Liver >> Spleen > Heart / Kidney >> Brain), so I feel confident that the observed data is correct. 

Thank you again for your help.
Shirley

**Comment by prvmalik** [comment]
Hi Shirley,

I assume you are modeling the disposition of a lysosomal enzyme bearing phosphomannosyl residues. Are you using the large molecule model or the small molecule model?

Phagocytosis is the main driver of their PK. Their 'apparent' fast plasma clearance is driven by phagocytic uptake into kupffer cells in the liver and then to a lesser extent, macrophages in the spleen. Kupffer cells line the sinusoids and are taking up the enzyme directly from the plasma, while in other organs the uptake is happening from the interstitial space. Further, the array data for M6PR in the GeneDB underestimates the effect of mannose-receptor bearing phagocytes in the liver since their expression is diluted across the whole organ volume.

In your particular model, the insensitivity to liver uptake is because:
1. Your plasma clearance is too fast and dominates the PK (but in reality, this plasma clearance is uptake and sequestration in kupffer cells, not protein hydrolysis)
2. The extraction ratio of the liver may already be quite high

I am working on a parameterization for macrophages in the large molecule model, if you can hold tight for 6 months...

Paul

**Comment by yishuanwu** [comment]
Thank you Paul, this is very insightful! 
Indeed I am using the large molecule model to model the disposition of a lysosomal enzyme with mannose 6 phosphate glycosylation. And indeed the fast plasma clearance is forced on by me using a hypothetical hydrolysis process. I saw that M6P receptors are intensely expressed in the hepatocytes but not in the Kupffer cells (Waguri 2002), and have not considered these other processes. 

I probably cannot wait 6 months, though I am still intimidated by MoBi. Any advice is very much appreciated. 

Shirley

**Comment by prvmalik** [comment]
Here's what I would do as a workaround:

Create large molecule model
Enter molecular weight, hydrodynamic radius (if known)

Create a dummy enzyme in the intracellular space of all organs to model intracellular clearance (Kdeg1). This is typically much slower than the plasma disappearance. -to be optimized later.

Set the endosomal uptake rate to zero. In Mobi, enter 'passive diffusion processes' to fudge the uptake.

1. Liver Plasma to Liver Intracellular (Kup1)
2. Organ Interstitial to Organ Intracellular (Kup2) x Macrophage/M6PR expression factor

Add plasma/interstitial hydrolysis to match the in vitro half-life if needed.

Then optimize Kdeg1, Kup1, Kup2. Inform the Macrophage/M6PR expression factors with literature where possible or optimize if not.

From my experience, these processes are easily saturable. If you have multiple dose levels, optimize michaelis-menten Km and Vmax for each process rather than assuming that they are first order.

**Comment by yishuanwu** [comment]
Thank you very much Paul. I am working on incorporating your advices.

**Comment by yishuanwu** [comment]
I ended up modeling the mice plasma and tissue concentrations as a TMDD process, in which receptor is located in the basolateral extracellular membrane and the drug-receptor complex is internalized from the interstitial side to the intracellular space. This describes my mice concentrations perfectly. I was pleased with the model as the fall in plasma concentration is mostly driven by drug internalization, and peptide hydrolysis is very small.

![image](https://user-images.githubusercontent.com/13445680/82834743-008de580-9e90-11ea-9393-7c13e426fae2.png)

But when I try to extrapolate the model to humans, my predicted concentrations were much higher (20x) than the human concentrations (I already checked that the data and dose are correct many many times). I try to increase the receptor concentration but that does not help. Overall, my model does respond to very large changes in the protein binding partner's expression, so I know I should have added the reactions successfully, and I repeated the steps for confirmation, but it does not appear to be as sensitive and the tissue uptake was much lower. 

![image](https://user-images.githubusercontent.com/13445680/82834763-0f749800-9e90-11ea-9e99-aeef5a681aa7.png)

I now want to try to separate the Mannose 6 Phosphate Receptor (interstitial --> intracellular) and the Macrophage Mannose Receptor (plasma --> intracellular), but I am having trouble modeling the latter. I first try to model it as a basolateral extracellular membrane protein and then try to tag the internalization process in the plasma space, but that creates an error. When 

I see there is a possibly related post inquiring about the difference between interstitial and extracellular membrane, and there is something about setting the localization in vasc. endothelium so that to add by plasma and interstitial expression:
https://github.com/Open-Systems-Pharmacology/Forum/issues/390
What should I do to model the uptake from plasma to inside the organ? 

I have been banging my head the past whole week. It's possible that humans have a bit more antibody formation, but what might be some other things I can try?

Thanks again, 
Shirley

**Comment by prvmalik** [comment]
The problem may be related to interspecies differences in interactions between the cell membrane, glycocalyx and the protein glycosylation patterns. 

As requested, to put in a Liver|Plasma to Liver|Intracellular process, take the following steps.

1. Create a neighborhood between Liver|Periportal|Plasma and Liver|Periportal|Intracellular compartments in the Organism's Spatial Structure.
![Step00](https://user-images.githubusercontent.com/29066045/82837381-dc81d280-9e96-11ea-8dc5-84f87c494f2b.JPG)


2. Add a parameter 'kup_liv' in the Organism's Spatial Structure parameters. I like to put the parameters here so that they are global. Favourite them for easy access.
![Step0](https://user-images.githubusercontent.com/29066045/82837421-fa4f3780-9e96-11ea-8ee0-4ad6a3691365.JPG)

3. Create a new passive transport process as shown:
![Step1](https://user-images.githubusercontent.com/29066045/82837506-41d5c380-9e97-11ea-864c-6686ae814372.JPG)

4. Create the kinetic equation by dragging the parameters from the right side of the navigator to the workspace. Get 'kup_liv' from the Organism. The path to the drug amount must be specific: "Liver|Periportal|Plasma|DrugName. Dragging the drug name gives the amount of the drug in the compartment. Expanding the drug name tab and clicking on the concentration gives the concentration of the drug in the compartment.
![Step4](https://user-images.githubusercontent.com/29066045/82837610-90835d80-9e97-11ea-8cac-0ffb2d6d6b5f.JPG)

If you want a Michaelis-Menten process, then add Km and Vmax as parameters in step 2 instead of kup_liv

**Comment by yishuanwu** [comment]
Thank you again Paul. 
This approach again could be used to model mice concentration well, but fails at extrapolation to humans. 
I will keep trying a few more things such as increasing the plasma to interstitial permeability or decreasing the enzyme radius (I have observed data for the drug's dimensions, my drug is not spherical -- the predicted radius is reasonably in the middle of the range of side lengths, but would the minimum side length be reasonable for use as the radius?), etc.

It seems to me that the human plasma concentration was very high because the drug cannot diffuse to the interstitial spaces fast enough to be internalized by TMDD. 
After incorporation of the passive diffusion process, Cmax is still overpredicted but then drops down very steeply. I think the diffusion process might have been too slow initially, and does not imitate the saturable active uptake transport later on so the enzyme is continuously draining from the plasma compartment. 

So I think I should wait for the macrophage add-on after all :) I will be excitedly looking for the add-on.

**Comment by yishuanwu** [comment]
I tried to circumvent the issue by creating neighborhoods of liver plasma to liver intracellular, and spleen plasma and spleen intracellular, separate plasma protein binding partner, and reactions in which the plasma drug-receptor complexes are internalized to liver/spleen intracellular. The drug does get taken up into the cell via this modification, but the liver/spleen concentrations could not be increased high enough, and when I tried to increase the number of the plasma binding partners or the rate of internalization, I got a paradoxical increase in heart/kidney concentrations. I have a hard time understanding why it may be so, so this approach to model the direct macrophage uptake from plasma was unsuccessful for me.
